src/services/camera/camera.service.ts

```diff
@@ -0,0 +1,219 @@
+import { Injectable } from '@angular/core';
+import { CesiumService } from '../cesium/cesium.service';
+import { SceneMode } from '../../models/scene-mode.enum';
+
+/**
+ *  The service exposes the scene's camera and screenSpaceCameraController
```

Comment:
add a small doc about the scene mode and how it affects tilt , rotate...

Reply:
done